tests/manage/test_pvresize_with_node_down.py

```diff
@@ -0,0 +1,228 @@
+"""
+OCS-368
+"""
+import logging
+import pytest
+import json
+from time import sleep
+import threading
+
+from ocs_ci.ocs import ocp, constants
+from ocs_ci.framework import config
+from ocs_ci.framework.testlib import tier1, ManageTest
+from ocs_ci.ocs.resources.pvc import PVC
+from ocs_ci.ocs.resources.pod import Pod
+from ocs_ci.ocs.resources.pod import get_ceph_tools_pod
+from ocs_ci.utility.utils import run_cmd
+from ocs_ci.ocs.exceptions import CommandFailed
+from ocs_ci.utility import templating
+
+log = logging.getLogger(__name__)
+
+OCS_BUG_ID = 'test-ocs-368'
+
+NAMESPACE = ocp.OCP(kind='namespace')
+OUR_PVC = None
+POD = None
+
+
+@pytest.fixture(scope='class')
+def test_fixture(request):
+    """
+    Finalize teardown and call setup
+    """
+    def finalizer():
+        teardown()
+    request.addfinalizer(finalizer)
+    setup()
+
+
+def check_obj(obj_type):
+    """
+    Check pvc and pod.  Only one of each will be created in this test.
+
+    Return metadata
+    """
+    cmd_str = f'oc -n {config.ENV_DATA["my_namespace"]} get {obj_type} -o json'
+    objs = json.loads(run_cmd(cmd_str))
+    assert len(objs['items']) == 1
+    log.info(f"{obj_type} created successfully.")
+    return objs['items'][0]['metadata']
+
+
+def setup():
+    """
+    Setting up the environment for the test
+
+    Create namespace
+    Create storageclass
+    Create pvc
+    Create pod
+    """
+    NAMESPACE.create(resource_name=OCS_BUG_ID)
+    config.ENV_DATA["my_namespace"] = OCS_BUG_ID
+
+    pvc_data = templating.load_yaml_to_dict(constants.CSI_PVC_YAML)
+    pvc_data['metadata']['namespace'] = config.ENV_DATA["my_namespace"]
+    pvc_data['spec']['storageClassName'] = 'rook-ceph-block'
+    pvc_data['spec']['resources']['requests']['storage'] = '100Gi'
+    pvc_name = pvc_data['metadata']['name']
+    global OUR_PVC
+    OUR_PVC = PVC(**pvc_data)
```

Comment:
use tests.helpers.create_pvc()

Reply:
Working on this.

Reply:
How can you set the storage to 100Gi and the namespace to something other than openshift-storage using helpers.create_pvc?

Reply:
Okay. I am working on changing the pvc function.